apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/profile/ProfileTaskExecutionService.java

```diff
@@ -151,7 +202,7 @@ public long getLastCommandCreateTime() {
      */
     private String checkProfileTaskSuccess(ProfileTask task) {
```

Comment:
In this case, I prefer to return a `Result` class, rather than exception. Profiling is happening in a high payload scenario, but Exception is a heavy OP. We should avoid it.

Reply:
> In this case, I prefer to return a `Result` class, rather than exception. Profiling is happening in a high payload scenario, but Exception is a heavy OP. We should avoid it.

Sounds good to me

Reply:
@kezhenxu94 @wu-sheng resolved, please check